rollup.config.js

```diff
@@ -53,16 +52,59 @@ const plugins = [
       'babel-plugin-transform-react-remove-prop-types',
     ],
   }),
-  url(),
-  svgr(),
+  url({
+    include: '**/inline-icons/*.svg',
+  }),
+  svgr({
+    include: '**/icons/*.svg',
+    titleProp: true,
+    svgo: true,
+    memo: true,
```

Comment:
Just curious to know the benefit of memoizing these SVG components.

Reply:
See #8313 for where I implemented this.